test/users.js

```diff
@@ -98,6 +98,27 @@ describe('Users', function() {
     }
   });
 
+  it('should verify a user password created with former credential package', async function() {
+    const req = apos.task.getReq();
+    const user = apos.user.newInstance();
+
+    user.title = 'Old User';
+    user.username = 'olduser';
+    user.password = 'passwordThatWentThroughOldCredentialPackageHashing';
+    user.email = 'old@user.com';
+    user.role = 'admin';
+
+    await apos.user.insert(req, user);
+
+    // A password hash that were generated by the former credential package:
+    const oldPasswordHashSimulated =
+      '{"hash":"/1GntJjtkMY1iPmQY1gn9f3bOZ5tb2qFL+x4qsDerZq2JL8+12TERR4/xqh246wBb+QJwwIRsF/6E+eccshsLxT/","salt":"GJHukLNaG6xDgdIpxVOpqV7xQLQM7e5xnhDW7oaUOe7mTicr7Ca76M4uUJalN/cQ68CE9O7yXZ5WJOz4RN/udcX0","keyLength":66,"hashMethod":"pbkdf2","iterations":2853053}';
```

Comment:
Editing userSafe with a passwordHash generated with the old `credential` package